types/types.go

```diff
@@ -153,6 +153,9 @@ type ImageDestination interface {
 	// uploaded to the image destination, true otherwise.
 	AcceptsForeignLayerURLs() bool
 
+	// AcceptsAnyOSSpecificImage returns true if the destination can store images for every operating system. False otherwise.
+	AcceptsAnyOSSpecificImage() bool
```

Comment:
… Actually explicitly saying “runtime OS” is somewhat nice, `AcceptsAny` says that everything is accepted, but when that is false, nothing says what is acceptable.  At least please say that in the comment which explains what `false` means.